main.js

```diff
@@ -629,10 +634,12 @@ function rmcourses(){
 	courses = [];
 }
 
-function idiot(){
-	rmcourses();
-	rmblocks();
-	courses = [];
+function idiot(isLoad = false){
+	if(!isLoad) {
```

Comment:
why is this?

Reply:
I forgot to finish implementing the skin change function properly apparently. Now it should make sense, with the latest commit.